app/controllers/sessions_controller.rb

```diff
@@ -7,7 +7,7 @@ def create
     if user
       session[:user_id] = user.id
       redirect_to root_path
-      flash[:success] = "Welcome to forsythia!"
+      flash[:success] = "Welcome to forsythia! Ready to #{view_context.link_to('work', '/start', class: 'text-strong')}?"
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long. [121/90]](https://www.codacy.com/app/conta-srdr/forsythia/file/2136964368/issues/source?bid=3139784&fileBranchId=3148684#l10)